src/clr-angular/forms/datepicker/date-container.spec.ts

```diff
@@ -115,6 +115,20 @@ export default function() {
         context.detectChanges();
       });
 
+      it('returns focus to the toggle button when closed with the escape key', () => {
+        const actionButton: HTMLButtonElement = context.clarityElement.querySelector('.clr-input-group-icon-action');
+        const actionButtonSpy = spyOn(actionButton, 'focus');
+        expect(actionButtonSpy.calls.count()).toBe(0);
+        actionButton.click();
+        context.detectChanges();
+        const event = new KeyboardEvent('keyup', {
+          key: 'Escape',
+        });
+        document.body.dispatchEvent(event);
+        context.detectChanges();
+        expect(actionButtonSpy.calls.count()).toBe(1);
+      });
+
```

Comment:
Shouldn't the test assert that the active element is back on the toggle button after pressing esc?

Reply:
I'm testing the the toggle button focus method was called after an escape event. It seemed the best way test that the toggle button had received focus.  

Is the suggestion the I should also add
```expect(document.activeElement).isEqual(actionButton);```?

Reply:
Ah, I overlooked the part that spies on the focus method. Thank you for clarifying. 

Yes, my suggestion would have been to  add something like `expect(document.activeElement).isEqual(actionButton);`. But I understand that this is a backport; I wouldn't block this PR on it.